helm/templates/ingress.yaml

```diff
@@ -21,8 +21,12 @@ metadata:
   name: {{ .Values.ingress.serviceName }}
 spec:
   rules:
+{{- if .Values.ingress.endpoint }}
   - host: {{ .Values.ingress.endpoint }}.{{.Values.ingress.server}}
     http:
+{{ else }}
+  - http:
+{{- end }}

```

Comment:
the first line after the rules must start with a `-`, hence the else

Reply:
This answers what would have been my comment then!